src/api/Intelligence.js

```diff
@@ -4,8 +4,13 @@
  * @author Box
  */
 
-import Base from './Base';
+import getProp from 'lodash/get';
 import type { BoxItem } from '../common/types/core';
+import { isUserCorrectableError } from '../utils/error.js';
+import Base from './Base';
+import { AiExtractResponse } from './schemas/AiExtractResponse.js';
+import { AiExtractStructured } from './schemas/AiExtractStructured.js';
+import { ERROR_CODE_EXTRACT_STRUCTURED } from '../constants';
```

Comment:
```suggestion
import { isUserCorrectableError } from '../utils/error';
import Base from './Base';
import { AiExtractResponse } from './schemas/AiExtractResponse';
import { AiExtractStructured } from './schemas/AiExtractStructured';
import { ERROR_CODE_EXTRACT_STRUCTURED } from '../constants';
```

You can drop the `.js` ext from the imports on all the files